packages/pipeline/src/scripts/pull_ohlcv_cryptocompare.ts

```diff
@@ -9,8 +9,10 @@ import { handleError } from '../utils';
 import { fetchOHLCVTradingPairsAsync, TradingPair } from '../utils/get_ohlcv_trading_pairs';
 
 const SOURCE_NAME = 'CryptoCompare';
+// tslint:disable:custom-no-magic-numbers
```

Comment:
Can you change this to `// tslint:disable-next-line:custom-no-magic-numbers`? That disables it for the next line only instead of the entire file.